src/components/artifacts/ArtifactsView.tsx

```diff
@@ -192,9 +207,14 @@ class ArtifactsView extends React.Component<Props> {
     return (
       <Paper elevation={1}>
         <Toolbar className={classes.title}>
-          <Typography variant="h6" color="inherit">
+          <Typography variant="h6" color="inherit" className={classes.title}>
             {this._currentPath() || 'Artifacts'}
           </Typography>
+          {this._getSelectedArtifact() === null ? null : (
+            <Button onClick={() => this.setState({ isFolderView: !this.state.isFolderView })}>
```

Comment:
I think here `selectedArtifactName` and `selectedPath` will be overriden. I think `updateState` should be used instead here.

Maybe also use `@material-ui/icons/ViewList` and `@material-ui/icons/AccountView` in a Toggle Button?

Reply:
Nice find! I'll fix this once I get home.